newdle/client/src/selectors.js

```diff
@@ -15,6 +15,7 @@ export const isAcquiringToken = state => !!state.auth.acquiringToken;
 
 // user
 export const getUserInfo = state => state.user;
+export const getUserTimezone = state => (state.user && state.user.timezone) || moment.tz.guess();
```

Comment:
Since we never have a user timezone in the state, i would only return the guessed one.

We can add it back if that changes at some point in the future.

Reply:
Sounds good, I did it just to show how we could receive it in the future, but I agree "code speculation" is confusing. I'll leave only moment.tz.guess